src/renderers/shared/stack/event/ReactSyntheticEventType.js

```diff
@@ -8,14 +8,29 @@
  *
  * Flow type for SyntheticEvent class that includes private properties
  *
- * @providesModule ReactSyntheticEvent
+ * @providesModule ReactSyntheticEventType
  * @flow
  */
 
 'use strict';
 
-export type DispatchConfig = any;
+import type {ReactInstance} from 'ReactInstanceType';
+
+export type DispatchConfig = {
+  dependencies: Array<string>,
+  phasedRegistrationNames?: {
```

Comment:
Good question - [this part of EventPluginRegistry](https://github.com/facebook/react/blob/9a88e593ed6f3a13f431048f0ee427255718ddbf/src/renderers/shared/stack/event/EventPluginRegistry.js#L108-L128) makes me think that the 'registrationName' property can be used when there is no 'phasedRegistrationNames' property. I will try to verify whether there are actually events created that follow that pattern.